ext/reflection/tests/readonly_class.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Readonly class reflection
+--FILE--
+<?php
+
+class Foo {
+}
+
+readonly class Bar {
+}
+
+$foo = new ReflectionClass(Foo::class);
+var_dump($foo->isReadOnly());
+var_dump(($foo->getModifiers() & ReflectionClass::IS_READONLY) != 0);
+
+$bar = new ReflectionClass(Bar::class);
+var_dump($foo->isReadOnly());
```

Comment:
I'm not pretty sure, but maybe you meanr `$bar` (not `$foo`) checks here?

Reply:
Thanks for the review, you are right!